<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_HeroSelection</class>
 <widget class="QWidget" name="Win_HeroSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,4,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Papyrus</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:#EEEEEE;
}</string>
     </property>
     <property name="text">
      <string>Nom de la partie</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="saveName">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{ border: 1px solid #888888; };</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="image">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonReturn">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLineEdit" name="heroName">
           <property name="font">
            <font>
             <family>Papyrus</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border:1px solid #888888;
	selection-background-color: #11FFFFFF;
	background-color:#11FFFFFF;
	color:#EEEEEE;
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Nom de votre héros</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="heroClass">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Papyrus</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
	background-color:#11FFFFFF;
	color: #EEEEEE;
	padding-left:90px;
	selection-background-color: #00FFFFFF;
}

QComboBox:editable {
   background-color:#11FFFFFF;
	color:#EEEEEE;
}


QComboBox:!editable, QComboBox::drop-down:editable {
    background-color:#11FFFFFF;
	color:#EEEEEE;
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background-color:#11FFFFFF;
	color:#EEEEEE;
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
	padding-left:90px;
}


QComboBox::drop-down {
    width: 0px;
    background-color:#00000000;
}


QComboBox QAbstractItemView {
	selection-background-color: #11FFFFFF;
    background-color:#11FFFFFF;
	color:#EEEEEE;
}</string>
           </property>
           <property name="currentText">
            <string>Epéiste</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="maxVisibleItems">
            <number>3</number>
           </property>
           <property name="maxCount">
            <number>10</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
           <property name="placeholderText">
            <string>Classe du héros</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Epéiste</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Archer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magicien</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="buttonContinue">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
       </property>
       <property name="text">
        <string>Continuer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
